src/Modal.js

```diff
@@ -320,7 +323,33 @@ const Modal = React.forwardRef(
       waitingForMouseUpRef.current = false;
     };
 
+    const handleStaticModalAnimation = () => {
+      setAnimateStaticModal(true);
+      removeStaticModalAnimationRef.current = transitionEnd(
```

Comment:
**suggestion:** we should probably cancel this on unmount, as well

Reply:
I currently cancel it in the `handleExit` function. Is that good enough?

Reply:
`onExit` won't be called for the component un-mounting, so the `setState` call at the end will fail

Reply:
Got it, thanks.